doc/en/components/charts/features/chart-axis-gridlines.md

```diff
@@ -22,45 +26,66 @@ Axis major gridlines are long lines that extend horizontally (y-axis) or vertica
 
+On category axes, this value is represented as fraction of the MajorInterval property. Generally, this value should equal to between 0.25 and 0.5 
 
-## Axis Tickmarks Example
+On numeric axes, this value is represented as double between axis minimum value and axis maximum value. By default, numeric axes will not automatically calculate minor interval based on axis minimum values and maximum value. 
 
-Axis tickmarks are short lines that extend from axis major gridlines towards axis labels. This example shows how to configure axis tickmark.
+On date time axes, this value is represented as time span between axis minimum value and axis maximum value. 
+
+The following example demonstrates how to customize the gridlines by setting the properties above:
 
 <code-view style="height: 450px"
            data-demos-base-url="{environment:dvDemosBaseUrl}"
-           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-axis-tickmarks"
-           alt="$PlatformShort$ Axis Tickmarks Example"
-           github-src="charts/category-chart/axis-tickmarks">
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-axis-gridlines"
+           alt="$PlatformShort$ Axis Gridlines Example"
+           github-src="charts/category-chart/axis-gridlines">
 </code-view>
 
 <div class="divider--half"></div>
 
-## Axis Gridlines Properties
+## $PlatformShort$ Axis Tickmarks
+
+Axis tick marks are enabled by setting the `TickLength` property of an axis to a value greater than 0. This property specifies the length of the line segments forming the tick marks.
+
+Tick marks are always displayed at the axis stroke line and point to the direction of the labels. Labels are offset by the value of the `TickLength` property to avoid overlapping. For example, for a horizontal axis where the label location is `AxisLabelsLocation.OutsideBottom`, if the `TickLength` is set to 5, labels will be shifted down by that amount.
 
-This table lists tickmark properties of axes that you can configure and the following example demonstrates how to set those gridline properties.
+You can customize how the axis tickmarks are displayed in your Blazor chat by setting the following properties: 
 
-Axis Visuals           | Type    | Property Names
------------------------|---------|-----------------------
-Tick Stroke Color      | string  | `XAxisTickStroke` <br> `YAxisTickStroke`
-Tick Stroke Thickness  | number  | `XAxisTickStrokeThickness` <br> `YAxisTickStrokeThickness`
-Tick Stroke Length     | number  | `XAxisTickLength` <br> `YAxisTickLength`
+- `TickLength`: This property sets the length of the axis tick marks.
+- `TickStrokeThickness`: This property sets the thickness of the axis tick marks.
+- `TickStroke`: This property sets the color of the tickmarks.
```

Comment:
these API members do not exist on CategoryChart and you cannot use API members of DataChart in topic that talks about CategoryChart. Please add back the table with correct API members and append a new column with description of those properties.